src/components/page-configuration/index.tsx

```diff
@@ -54,6 +56,21 @@ export interface PageConfigurationProps
   appName: string;
 }
 
+function getConfigBranchUrl({
+  configBranch,
+  repository,
+}: ApplicationRegistrationModel): string {
+  return `${repository}/tree/${configBranch}`;
+}
+
+function getConfigFileUrl({
+  configBranch,
+  radixConfigFullName,
+  repository,
+}: ApplicationRegistrationModel): string {
+  return `${repository}/blob/${configBranch}${radixConfigFullName}`;
```

Comment:
oops, missed a slash
```suggestion
  return `${repository}/blob/${configBranch}/${radixConfigFullName}`;
```